exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -46,6 +47,14 @@ var (
 	reInvalidSpanCharacters = regexp.MustCompile(`[^ 0-9\p{L}N_.:/%&#=+,\-@]`)
 )
 
+var (
+	remoteXrayExporterDotConverter = featuregate.GlobalRegistry().MustRegister(
```

Comment:
I believe there is a featuregate.FromVersion() optional function. Can you please apply it to this feature gate and use collector contrib version + 1 as the value.

Reply:
Done